.github/workflows/linux_meson.yml

```diff
@@ -290,7 +290,7 @@ jobs:
       run: |
         # TODO: unpin cython and move to -pre once issues with cython 3.0
         #       have been ironed out.
-        python -m pip install ninja meson meson-python wheel click rich_click pydevtool "cython==0.29.33"
+        python -m pip install ninja meson meson-python wheel click rich_click pydevtool "git+https://github.com/matusvalo/cython.git@struct_visibility"
```

Comment:
Better for the cython to have its own line?

Reply:
> Better for the cython to have its own line?

I suppose yes, but right now my main focus is to get scipy in working state so I am not paying attention to formatting.